src/test/java/duke/DeadlineTest.java

```diff
@@ -24,8 +27,8 @@ void getRawDateTimeTest() {
         LocalDateTime today = LocalDateTime.now();
```

Comment:
Test methods could be named as dateTimeTest_RawDateTime (following `featureUnderTest_testScenario`).